docs/api.rst

```diff
@@ -12408,3 +12408,36 @@ Get avatar for an ens name
    :statuscode 400: Invalid ens name provided (it doesn't end with `.eth`).
    :statuscode 404: There is no avatar set for the given ens name.
    :statuscode 409: Avatar was requested for an ens name that is not currently in the database or we couldn't query the blockchain.
+
+
+Clear Icons/Avatars Cache
+============================
+
+.. http:post:: /api/(version)/cache/images/<cache_type>/clear
```

Comment:
If you are adding a new `/cache` endpoint perhaps it would make sense to move other cache/data cleaning there?